CMakeLists.txt

```diff
@@ -230,8 +168,22 @@ add_executable(llvm2bpl
   tools/llvm2bpl/llvm2bpl.cpp
 )
 
+# We have to import LLVM's cmake definitions to build sea-dsa
+# Borrowed from sea-dsa's CMakeLists.txt
+find_package (LLVM 8.0.1 EXACT CONFIG NO_DEFAULT_PATH)
```

Comment:
We are we specifying `NO_DEFAULT_PATH`? This seems to disallow pre-existing LLVM installations to be used. When running CMake with this parameter I see the following (on macOS with llvm@8 from Homebrew):
```console
% (rm -r build ; mkdir build && cd build && cmake .. -G Ninja)
rm: build: No such file or directory
-- The C compiler identification is AppleClang 11.0.0.11000033
-- The CXX compiler identification is AppleClang 11.0.0.11000033
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find LLVM (missing: LLVM_DIR)
CMake Error at CMakeLists.txt:175 (include):
  include could not find load file:

    AddLLVM


CMake Error at CMakeLists.txt:176 (include):
  include could not find load file:

    HandleLLVMOptions


CMake Error at sea-dsa/src/CMakeLists.txt:1 (add_llvm_library):
  Unknown CMake command "add_llvm_library".


-- Configuring incomplete, errors occurred!
See also "/Users/mje/Code/smack/build/CMakeFiles/CMakeOutput.log".
```
Without this parameter CMake succeeds.